packages/opentelemetry-sdk-trace-node/package.json

```diff
@@ -59,7 +59,7 @@
     "typescript": "4.4.4"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.1.0"
+    "@opentelemetry/api": ">=1.0.0 <1.2.0"
```

Comment:
Thinking one more about this now with https://github.com/open-telemetry/opentelemetry-js/issues/2755#issuecomment-1028352909 in mind it seems to be not that good to allow more then one minor in SDK.

If SDK ends up in using 1.0.x because the de-duplication of package manager decide so but other components use 1.1.x these other components will have issues as the version check in API signals incompatibility.

I guess the SDK must use the highest minor.